deployment/network-operator/values.yaml

```diff
@@ -158,6 +158,12 @@ sriovDevicePlugin:
   resources:
     - name: hostdev
       vendors: [15b3]
+    - name: infiniband_rdma
```

Comment:
This affects the default configuration right ? i dont think we want that.

also these 3 resources now collide 

1 includes 2 and 2 
2 may include 3

Reply:
Let's keep the default configuration as is and only change the examples in readme